docs/releases-notes/keycloak.md

```diff
@@ -1,5 +1,37 @@
 # Keycloak
 
+#### **Keycloak v4.8.0**
+
+* https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-4-8-0-final
+
+#### **User Required Actions**
+
+Post release of Keycloak v4.6.0 the `aud` claim was removed from the generated access token. This causes an issue specifically for those using the keycloak-proxy / keycloak-gatekeeper which uses go-oidc (github.com/coreos/go-oidc) under the hood as it validates it's existence.
+
+The easiest way to mitigate the issue is add the `aud` claim either in the Client scopes or on the keycloak client you're using, i.e.
+
+* Go to realm settings
+* Click on the client your using to login with and go to "Mappers" tab
+* Click "Create" and add
+  * Name: Audience
+  * Mapper Type: Hardcoded Claim
+  * Token Claim Name: aud
+  * Claim Value: <the name of the this client>
```

Comment:
The `<..>` has been omitted as it's not wrapped in backticks. Or you could put this whole section in three backticks

Reply:
`the this client` => `this client`